src/pages/library/common.js

```diff
@@ -16,6 +17,8 @@ import * as Nav from 'src/libs/nav'
 import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 
+import { RequestDatasetAccessModal } from './RequestDatasetAccessModal'
```

Comment:
Can you convert this to not use a relative path?

Reply:
Good idea, sure